// Copyright (C) 2020 THL A29 Limited, a Tencent company. All rights reserved.
//
// Licensed under the BSD 3-Clause License (the "License"); you may not use this
// file except in compliance with the License. You may obtain a copy of the
// License at
//
// https://opensource.org/licenses/BSD-3-Clause
//
// Unless required by applicable law or agreed to in writing, software
// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
// License for the specific language governing permissions and limitations under
// the License.

syntax = "proto3";

package flare.binlog.text_only;

option go_package = "tencent.com/protobuf/flare/rpc/binlog/text_only/binlog_proto";

import "flare/rpc/binlog/util/proto_binlog.proto";

message Log {
  proto.Call incoming_call = 1;
  repeated proto.Call outgoing_calls = 2;
};
